actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -12,16 +12,16 @@ def initialize(*)
 
       def broadcast(channel, payload)
         with_connection do |pg_conn|
-          pg_conn.exec("NOTIFY #{pg_conn.escape_identifier(channel)}, '#{pg_conn.escape_string(payload)}'")
+          pg_conn.exec("NOTIFY #{pg_conn.escape_identifier(channel.to_s)}, '#{pg_conn.escape_string(payload)}'")
```

Comment:
http://ruby-doc.org/core-2.3.0/Kernel.html#method-i-String
> First tries to call its `to_str` method, then its `to_s` method.
But apparently it doesn't fail if implicit conversion fails.